xcube/webapi/datasets/config.py

```diff
@@ -238,6 +240,14 @@
     additional_properties=True,
 )
 
+ENTRYPOINT_DATASET_ID_SCHEMA = JsonObjectSchema(
+    properties=dict(
+        Identifier=IDENTIFIER_SCHEMA,
+    ),
+    required=["Identifier"],
+    additional_properties=False,
+)
+
```

Comment:
Why isn't this a string? The field is already called `EntrypointDatasetId`. With `Id` in it.